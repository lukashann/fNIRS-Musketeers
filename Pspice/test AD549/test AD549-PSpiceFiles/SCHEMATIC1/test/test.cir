** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test AD549\test AD549-PSpiceFiles\SCHEMATIC1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "H:/Uni Stuff/Year 3/Project/Pspice/models/opa124_pspice_aio/opa124.lib" 
.LIB "H:/Uni Stuff/Year 3/Project/Pspice/models/ad549/ad549.lib" 
* From [PSPICE NETLIST] section of H:\cdssetup\OrCAD_PSpice/16.6.0/PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.DC LIN I_I1 0 100p 0.001p 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
