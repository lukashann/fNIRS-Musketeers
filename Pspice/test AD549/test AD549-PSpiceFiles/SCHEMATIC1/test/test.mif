 lib="H:\Uni Stuff\Year 3\Project\Pspice\models\opa124_pspice_aio\opa124.lib", offset=1790, size=1441
 lib=C:\Cadence\SPB_16.6\tools\PSpice\Library\opamp.lib, offset=54585, size=371
 lib=C:\Cadence\SPB_16.6\tools\PSpice\Library\opamp.lib, offset=53165, size=1419
