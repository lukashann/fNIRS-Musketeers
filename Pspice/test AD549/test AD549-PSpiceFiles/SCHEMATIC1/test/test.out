
**** 11/18/16 14:35:58 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test AD549\test AD549-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "H:/Uni Stuff/Year 3/Project/Pspice/models/opa124_pspice_aio/opa124.lib" 
.LIB "H:/Uni Stuff/Year 3/Project/Pspice/models/ad549/ad549.lib" 
* From [PSPICE NETLIST] section of H:\cdssetup\OrCAD_PSpice/16.6.0/PSpice.ini file:
.lib "nom.lib" 

*Analysis directives: 
.DC LIN I_I1 0 100p 0.001p 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source TEST AD549
V_V1         N00683 0 5Vdc
V_V2         0 N00200 5Vdc
I_I1         N00183 N00176 DC 10pAdc  
C_C1         N00183 N00395  100p  TC=0,0 
R_R1         N00183 N00395  10G TC=0,0 
X_U2         N00395 N01083 N01152 N01227 N01083 LM358 
V_V3         N01152 0 5Vdc
V_V4         0 N01227 5Vdc
X_U3         N00183 N00176 N00683 N00200 N00395 OPA124

**** RESUMING test.cir ****
.END

INFO(ORPSIM-15423): Unable to find index file opa124.ind for library file opa124.lib.

INFO(ORPSIM-15422): Making new index file opa124.ind for library file opa124.lib.

Index has 1 entries from 1 file(s).

**** 11/18/16 14:35:58 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test AD549\test AD549-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_U3.DY         X_U3.DX         X_U2.x_lm358.dx X_U2.x_lm358.dy 
          IS  380.000000E-18  800.000000E-18  800.000000E-18  800.000000E-18 
          RS                                    1               1.000000E-03 
         CJO                                                   10.000000E-12 
          KF  117.500000E-18                                                 


**** 11/18/16 14:35:58 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test AD549\test AD549-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     BJT MODEL PARAMETERS


******************************************************************************




               X_U2.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


**** 11/18/16 14:35:58 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test AD549\test AD549-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     Junction FET MODEL PARAMETERS


******************************************************************************




               X_U3.JX         
               PJF             
         VTO   -1            
        BETA     .01579      
          IS  400.000000E-15 


**** 11/18/16 14:35:58 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test AD549\test AD549-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     Current Controlled Switch MODEL PARAMETERS


******************************************************************************




               X_U2.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U2.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U2.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U2.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 



          JOB CONCLUDED

**** 11/18/16 14:35:58 ***** PSpice 16.6.0 (October 2012) ***** ID# 0 ********

 ** Profile: "SCHEMATIC1-test"  [ H:\Uni Stuff\Year 3\Project\Pspice\test AD549\test AD549-PSpiceFiles\SCHEMATIC1\test.sim ] 


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =        4.23
